packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -84,11 +85,32 @@ export async function tileSetToStyle(
     `/v1/tiles/${tileSet.name}/${tileMatrix.identifier}/{z}/{x}/{y}.${tileFormat}${query}`;
 
   const styleId = `basemaps-${tileSet.name}`;
-  const style = {
-    version: 8,
-    sources: { [styleId]: { type: 'raster', tiles: [tileUrl], tileSize: 256 } },
-    layers: [{ id: styleId, type: 'raster', source: styleId }],
-  };
+  let style;
+
+  if (tsElevation) {
+    const elevationUrl =
+    (Env.get(Env.PublicUrlBase) ?? '') +
+    `/v1/tiles/${tsElevation.name}/${tileMatrix.identifier}/{z}/{x}/{y}.${tileFormat}${query}&pipeline=terrain-rgb`;
+    style = {
```

Comment:
We should use `tileSet.outputs` to determine the sources to and how they are layered

we can hard code terrain-rgb to raster-dem and color-ramp to raster.